app/models/user.rb

```diff
@@ -83,4 +86,20 @@ def self.search(search)
   def name
     "#{firstname} #{lastname}"
   end
+
+  private
+
+=begin
```

Comment:
what does the '=' mean?

Reply:
It is the begining of a multiline comment

Reply:
thanks